src/NServiceBus.Core/Persistence/InMemory/Outbox/InMemoryOutboxPersistence.cs

```diff
@@ -65,7 +65,7 @@ void PerformCleanup(object state)
             readonly InMemoryOutboxStorage inMemoryOutboxStorage;
             readonly TimeSpan timeToKeepDeduplicationData;
 
-// ReSharper disable once NotAccessedField.Local
+            // ReSharper disable once NotAccessedField.Local
```

Comment:
Should this be replaced with `#pragma` just as done in other files? Or this R# specific comment is not longer relevant.

Reply:
Since we don't do R# inspections in CI anymore, none of the R# comments are relevant. I think we've removed all of them in `master` branches, but it looks like they are still around in the release branches.

This change was just a formatting change, and not actually related to the comment contents.